api.planx.uk/modules/send/routes.ts

```diff
@@ -16,6 +16,7 @@ router.post(
   createSendEvents,
 );
 router.post("/bops/:localAuthority", useHasuraAuth, sendToBOPS);
+router.post("/bops-v2/:localAuthority", useHasuraAuth, sendToBOPSV2);
```

Comment:
Open to naming feedback here; also considered `/bops-odp` ? 

But ultimately, I'm hoping our overlap period for sending both is quite short & the new endpoint will simply be renamed soon to replace the existing `/bops` one. Since we only use our send routes internally, I figure renaming isn't a particular pain or inconvenience. 

Both endpoints write to the exisiting audit table `bops_applications` so no changes there.

Reply:
Naming convention here makes sense to me given it's - 
 - short term
 - can be easily updated as it's really just an internal ref

Reply:
If we made `v2` a param it might be simpler to just re-use the existing function and pump in `"v2"` where needed but as this is just a short-lived thing I'm totally happy with copy/pasting the function tbh!

On the off chance that the original BOPS code will need to be updated in the next few weeks, we'll just need to remember to update both.